### PR TITLE
Negative feedback comes from a good place

### PR DESCRIPTION
**Pull request by znck**

### CODE REVIEW

File: src/blog/articles/2020-appreciate-feedback/readme.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+It's not easy to say negative things to people. In fact, it takes a lot of
+courage to point out shortcoming. If someone does that for me, honest and
+brutal, I keep them close. Brutal and honest friends are dearest to me.
+
+<!-- more -->
+
+## Feedback is hard
+
+We used to have 360-degree feedbacks at Myntra (I mean they still have, I am no
+more with them but still we). I used to hate those and often treated them waste
```
Comment (swapagarwal) [inline src/blog/articles/2020-appreciate-feedback/readme.md]: but still we... seems incomplete.

Comment (swapagarwal) [reply src/blog/articles/2020-appreciate-feedback/readme.md]: * a waste

File: src/blog/articles/2020-appreciate-feedback/readme.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+of time. When I look back today, I hated because I couldn't write about people
+and their work. I couldn't write because it's very difficult to recall what they
+did this year and how could I remember their work when I was having hard time
```
Comment (swapagarwal) [inline src/blog/articles/2020-appreciate-feedback/readme.md]: \* a hard

File: src/blog/articles/2020-appreciate-feedback/readme.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+remembering mine. What did I do this year? I have been working on this component
+past few weeks. And oh! I shipped that feature before that. And... and...
```
Comment (swapagarwal) [inline src/blog/articles/2020-appreciate-feedback/readme.md]: \* for the past

File: src/blog/articles/2020-appreciate-feedback/readme.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+I used to face the same empty wall when I had to ask for feedback. We were
+living in now with no recollection of past, very little recollection. This wasn't
+working for me and I guess neither for people working with me, I feel they even
+didn't know that we weren't a good team. The first 360-feedback was a disaster, I
+didn't write anything, about me or my work or them or their work, nothing at all.
+
+My manager called for an 1-on-1, "Why didn't you fill your self-review?". I
+didn't tell him that I didn't remember, "Do we really need to? I mean I working
+well, you know that. Isn't it a waste of time?". That day I got to know that
+360-feedbacks are precursor to the forthcoming calibration meeting, unknown to
```
Comment (swapagarwal) [inline src/blog/articles/2020-appreciate-feedback/readme.md]: \* a precursor

File: src/blog/articles/2020-appreciate-feedback/readme.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+me, these corporate shenanigans, I thought. The calibration meeting was a
+hidden championship where every employee was compared with every other in same
```
Comment (swapagarwal) [inline src/blog/articles/2020-appreciate-feedback/readme.md]: \* the same

File: src/blog/articles/2020-appreciate-feedback/readme.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+role. Hidden because you don't represent yourself, instead your manager
+represents you. "But, you know what I have worked on", it struck me, shit—what
+the hell would he remember about my work when I couldn't. Maybe subconsciously I
+thought he's oracle or something, he would know, I can count on him, go be my
+champion and fetch me a promotion. But, it doesn't work like that and I am lucky
+I got know this very early in my career.
```
Comment (swapagarwal) [inline src/blog/articles/2020-appreciate-feedback/readme.md]: \* to know

File: src/blog/articles/2020-appreciate-feedback/readme.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+
+## It takes courage
+
+I started writing all major task I was working or peers who interacted with
```
Comment (swapagarwal) [inline src/blog/articles/2020-appreciate-feedback/readme.md]: \* tasks

File: src/blog/articles/2020-appreciate-feedback/readme.md [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+did keep a note of everything important. It helped me in 1-on-1s to set my
+goals, to steer myself into a kind of position I wanted.
+
+Today when I look back, I feel I didn't do justice to my peers. I didn't tell
+them in time, I waited for the 360-feedback because I had content, I could write
+this time and I did write. But that feedback was useless, it was late by 6
+months, they couldn't do anything then, even if they eagerly wanted to do
+something. I set them up for failure. I did. But what else could I do, I didn't
+know you're supposed to tell people, nobody told me. Nobody asked me for
+feedback, no before the D-day. And I didn't have the courage to tell them
```
Comment (swapagarwal) [inline src/blog/articles/2020-appreciate-feedback/readme.md]: \* not before

File: src/blog/articles/2020-appreciate-feedback/readme.md [hunk 9]
```diff
@@ -0,0 +1,210 @@
...
+unasked for. I takes courage to give feedback, even more when unasked for.
```
Comment (swapagarwal) [inline src/blog/articles/2020-appreciate-feedback/readme.md]: \* It takes

File: src/blog/articles/2020-appreciate-feedback/readme.md [hunk 10]
```diff
@@ -0,0 +1,210 @@
...
+feedback. And keep telling your head, she is trying to help me, be rational, be
+reasonable, open to suggestions and don't shut her off, listen, listen, listen
+and ask.
+
+## Not taking personally is a superpower
+
+"That's not a good way to do it, it could turn into some serious performance
+issue".
+
+We attach ourselves to our work so much that tend to loose where do we
```
Comment (swapagarwal) [inline src/blog/articles/2020-appreciate-feedback/readme.md]: \* lose

File: src/blog/articles/2020-appreciate-feedback/readme.md [hunk 11]
```diff
@@ -0,0 +1,210 @@
...
+
+### Ask for feedback
+
+You won't get anything unless you ask—promotion, relationship, and even feedback,
+so ask, ask and ask.
+
+Who do you ask? Ask your manager, ask your peers, ask your friends, ask your
+partner, ask your parents, ask that person who listened you talk, ask that
+person who read your article, ask that maintainer who reviewed your pull request,
+ask that that person who depends on your work or your work depends on, ask
```
Comment (swapagarwal) [inline src/blog/articles/2020-appreciate-feedback/readme.md]: double "that"s

File: src/blog/articles/2020-appreciate-feedback/readme.md [hunk 12]
```diff
@@ -0,0 +1,210 @@
...
+hard to share failures but failures need to be heard, failures make us better.
+We hear way less about failures even though they are more common, everyone fails
+at something or sometimes. If my suggestions does not work well for you, I expect
+to hear back, I don't want to set other people on wrong path, I want to hear I
+failed, I want to learn, don't take away my failures.
+
+What if you already know it won't work because you've tried or heard about it?
+You still report back, if someone is courageous enough to give you feedback, they
+surely can handle feedback or I'll assume they can. Do not hide failures and do
+share succusses, we need to hear them all.
```
Comment (swapagarwal) [inline src/blog/articles/2020-appreciate-feedback/readme.md]: \* successes

File: src/blog/articles/2020-appreciate-feedback/readme.md (hunk 10 above)
Comment (znck) [reply src/blog/articles/2020-appreciate-feedback/readme.md]: I think `to loose` is correct here

Comment (swapagarwal) [reply src/blog/articles/2020-appreciate-feedback/readme.md]: https://www.grammarly.com/blog/loose-lose/

**Pull request merged**